testnets/celestiatestnet2/chain.json

```diff
@@ -26,30 +26,16 @@
   },
   "codebase": {
     "git_repo": "https://github.com/celestiaorg/celestia-app",
-    "recommended_version": "v1.11.0",
-    "compatible_versions": [
-      "v1.3.0",
-      "v1.6.0",
-      "v1.7.0",
-      "v1.9.0",
-      "v1.10.1",
-      "v1.11.0"
-    ],
+    "recommended_version": "v2.0.0",
+    "compatible_versions": ["v2.0.0", "v2.1.0"],
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/celestiaorg/networks/master/arabica-11/genesis.json"
     },
     "versions": [
       {
-        "name": "v1.11.0",
-        "recommended_version": "v1.11.0",
-        "compatible_versions": [
-          "v1.3.0",
-          "v1.6.0",
-          "v1.7.0",
-          "v1.9.0",
-          "v1.10.1",
-          "v1.11.0"
```

Comment:
Note that, within 'versions', it is best to retain the version history (so v2 would be added as a new object below this v1 object). Perhaps I won't require it for a testnet, but this is the intention.

Reply:
Even with a network upgrade? https://blog.celestia.org/lemongrass-celestias-first/

Reply:
oh, is it a hardfork?

Reply:
yes, and will be referring to it as a "network upgrade" going forward (see https://github.com/celestiaorg/docs/issues/1676 for context)

Reply:
that terminology would be confusing here because a network upgrade generally means the version number increases but the genesis file stays the same. Shouldn't this hardfork (or "Network upgrade" as you call it) has a new genesis file too?

Reply:
the genesis file does stay the same. the chain-id also stays the same. you just can't run a consensus node with v1.x.x software, because it is not compatible with v2.x.x software, hence dropping the array of v1.x.x. versions. does that help?

Reply:
I don't know about running a consensus node, but this sounds like a standard version upgrade (and NOT a new network), in which case I think it makes sense to retain the version history. Note that, even though v1 would be included in the version history, only the versions listed under compatible_versions (and the recommended_version) are actually compatible with the current blocks.

Reply:
ahh, that makes a ton of sense. thanks for clarifying @JeremyParish69 !

Reply:
where should V1 be included in version history?

Reply:
Thanks for all the discussion on this! I updated this file with the historical v1.x releases based on how cosmoshub/chain.json did it. Please LMK if anything looks incorrect.

Reply:
LGTM, thank you @rootulp and @JeremyParish69 !